TFT/src/User/Menu/Touchmi.c

```diff
@@ -0,0 +1,61 @@
+#include "includes.h"
+#include "coordinate.h"
+
+const MENUITEMS autoLevelingItems = {
```

Comment:
this name is already being used in ABL.c

Reply:
Ok, i changed this name. thx